airbyte-protocol/protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -452,16 +452,6 @@ definitions:
       supportsIncremental:
         description: (deprecated) If the connector supports incremental mode or not.
         type: boolean
-      # Destination Connectors Properties
-      # Normalization is currently implemented using dbt, so it requires `supportsDBT` to be true for this to be true.
-      supportsNormalization:
-        description: If the connector supports normalization or not.
-        type: boolean
-        default: false
-      supportsDBT:
-        description: If the connector supports DBT or not.
-        type: boolean
-        default: false
```

Comment:
There are probably some related updates to make in https://github.com/airbytehq/airbyte/blob/master/docs/understanding-airbyte/airbyte-protocol.md#actor-specification

Reply:
Done, thanks for the suggestion.